src/lwc/geFormRenderer/geFormRendererHelper.js

```diff
@@ -89,3 +89,53 @@ function createSimplifiedObjectForObjectDevName(targetFieldApiNameBySourceFieldA
     return simplifiedObjectForObjectDevName;
 }
 
+
+export class GeFormElementHelper {
```

Comment:
@psadie-sfdo What do you think about this location?  Should GeFormElementHelper be its own standalone file?

Reply:
I think this is doing enough that I'd prefer to have it in a standalone file as before. Is there a big pro I'm maybe not seeing to keeping it here?